rust/connlib/tunnel/src/client.rs

```diff
@@ -379,7 +379,9 @@ impl ClientState {
                         }
                     })
                     .unwrap_or_else(|e| {
-                        tracing::debug!(error = std_dyn_err(&e), "Recursive UDP DNS query failed");
+                        let error = std_dyn_err(&e);
+                        tracing::debug!(error, "Recursive UDP DNS query failed");
+                        tracing::trace!(target: "telemetry", error, "Recursive UDP DNS query failed");
```

Comment:
Why have a lower level for telemetry?

Reply:
Locally and when debugging in general, I like to set `RUST_LOG=debug` and I didn't the telemetry events to show up there.

Reply:
I kind of see this similar as to `wire=trace`. They should really never show up in any logs unless you explicitly enable it. And if somebody is running our app with `RUST_LOG=trace` then they are hopefully aware that this will produce A LOT of logs :D